archimedes/archimedes/runner.py

```diff
@@ -52,6 +52,21 @@ class RunRequest:
     environment: List[str] = field(default_factory=list)
     image: str = "archimedes"
     isolator: str = "docker"
+    interpreter: str = "python"
+
+    def run_with(self, interp: str):
```

Comment:
Some helper methods on the request object to match file extensions to the interpreter.